src/commands/coffeeChat/coffeeChat.ts

```diff
@@ -1,36 +1,34 @@
 import { CodeyUserError } from './../../codeyUserError';
 // Sapphire Specific:
 // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-import { ApplyOptions } from '@sapphire/decorators';
-// Sapphire Specific:
-// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
 import { Args, container } from '@sapphire/framework';
 // Sapphire Specific:
 // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-import {
-  SubCommandPluginCommand,
-  // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-  SubCommandPluginCommandOptions,
-} from '@sapphire/plugin-subcommands';
-import { Message, MessageEmbed } from 'discord.js';
+import { Subcommand } from '@sapphire/plugin-subcommands';
+import { Message, EmbedBuilder, PermissionsBitField } from 'discord.js';
 import { getMatch, testPerformance, writeHistoricMatches } from '../../components/coffeeChat';
 import { DEFAULT_EMBED_COLOUR } from '../../utils/embeds';
 import { alertMatches } from '../../components/coffeeChat';
 
-@ApplyOptions<SubCommandPluginCommandOptions>({
-  aliases: ['coffee'],
-  description: 'Handle coffee chat functions.',
-  detailedDescription: `**Examples:**
+export class CoffeeChatCommand extends Subcommand {
```

Comment:
First example of new Sapphire command system. Instead of decorators we create a child constructor which passes parameters